src/tests/test_weighted_graph.py

```diff
@@ -0,0 +1,244 @@
+"""Test for our simple graph module."""
+
+import pytest
+
+
+@pytest.fixture
+def empty_graph():
+    """Instantiate an empty graph."""
+    from weighted_graph import Graph
+    g = Graph()
+    return g
+
+
+@pytest.fixture
+def graph_no_edge(empty_graph):
+    """Instantiate a graphe with nodes and no edge."""
```

Comment:
Interesting spelling of graph.  Alternatively, you misspelled every other instance of graphe.